browser/ui/views/split_view/split_view_browsertest.cc

```diff
@@ -72,17 +72,9 @@ ui::MouseEvent GetDummyEvent() {
 }
 }  // namespace
 
-class SplitViewDisabledBrowserTest : public InProcessBrowserTest {
- public:
-  SplitViewDisabledBrowserTest() {
-    scoped_features_.InitAndDisableFeature(tabs::features::kBraveSplitView);
-  }
-  ~SplitViewDisabledBrowserTest() override = default;
-
- private:
-  base::test::ScopedFeatureList scoped_features_;
-};
+using SplitViewDisabledBrowserTest = InProcessBrowserTest;
```

Comment:
`InProcessBrowserTest` should only be used for things that will always be desktop only, so just flagging this here to be sure.